config/initializers/subscriptions.rb

```diff
@@ -13,22 +10,23 @@
 
 # Ingestion/Creation
 ActiveSupport::Notifications.subscribe(Ddr::Notifications::CREATION, Ddr::Events::CreationEvent)
```

Comment:
Is this still needed?  What's the relationship between this Ddr::Notifications::CREATION notification and the Ddr::Models::Base::INGEST notification?  Backwards compatibility?  Same question applies to update and deletion notifications below.